README.md

```diff
@@ -12,8 +12,8 @@ and store structured log data, including:
  archives
 * The [zapi](cmd/zapi/README.md) command line tool, for interacting with the
 API provided by zqd
-* The [ZQL](zql/docs/README.md) query language definition and implementation
-* The [ZNG](zng/docs/README.md) structured log specification and supporting components
+* The [ZQL](docs/language/README.md) query language definition and implementation
+* The [ZNG](docs/formats/README.md) structured log specification and supporting components
```

Comment:
```suggestion
* The [ZNG](docs/formats/spec.md) structured log specification and supporting components
```

Reply:
I realized spec.md should be zng.md so I will change it throughout...